### PR TITLE
674909 create nested project

### PR DESCRIPTION
**Pull request by jimbodriven**
This was originally submitted by @kometes. We rebased on the development branch and resubmitted the pull request to development.

### CODE REVIEW

File: test/test_project.py [hunk 1]
```diff
@@ -72,10 +82,10 @@ def test_update_copy_fields(self):
             m.put(self.baseurl + '/1d0304cd-3796-429f-b815-7258370b9b74', text=response_xml)
             single_project = TSC.ProjectItem('test')
             single_project._id = '1d0304cd-3796-429f-b815-7258370b9b74'
-            single_project._permissions = 'Test to check if permissions copied over.'
+            single_project._some_field = 'Test to check if fields are copied over.'
```
Comment (t8y8) [inline test/test_project.py]: Why this change?

`_some_field` isn't an attribute of the model, actually, neither is `_permissions`. 

I think you can just delete this from the test entirely

Comment (jimbodriven) [inline test/test_project.py]: We were confused by this as well, at first. The whole point of the test is to verify that attributes not defined in the class definition get copied over to the new object. We update the name to make it more obvious.

File: samples/create_project.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    except TSC.ServerResponseError:
+        print('We have already created this project: %s' % project_item.name)
+        sys.exit()
+
+def main():
+    parser = argparse.ArgumentParser(description='Get all of the refresh tasks available on a server')
+    parser.add_argument('--server', '-s', required=True, help='server address')
+    parser.add_argument('--username', '-u', required=True, help='username to sign into server')
+    parser.add_argument('--site', '-S', default=None)
+    parser.add_argument('-p', default=None)
```
Comment (jimbodriven) [inline samples/create_project.py]: Sure thing.

File: samples/create_project.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+
+def create_project(server, project_item):
+    try:
+        project_item = server.projects.create(project_item)
+        print('Created a new project called: %s' % project_item.name)
+        return project_item
+    except TSC.ServerResponseError:
+        print('We have already created this project: %s' % project_item.name)
+        sys.exit()
```
Comment (jimbodriven) [inline samples/create_project.py]: Nods.
